### PR TITLE
Create 667A - Pouring Rain

### PR DESCRIPTION
**Pull request by Supporter09**
Miss 667A so i added it ( AC tested on codeforces )

### CODE REVIEW

File: A-set/667A. Pouring Rain.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import math
+d,h,v,e = map(int, input().split())
```
Comment (Waqar-107) [inline A-set/667A. Pouring Rain.py:2]: Please add a new line after import. when you are declaring the variables, it's a good practice to use a space after the comma. Like, `d, h, v, e`

File: A-set/667A. Pouring Rain.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import math
+d,h,v,e = map(int, input().split())
+drink_s = (4*v)/(math.pi * (d**2))
```
Comment (Waqar-107) [inline A-set/667A. Pouring Rain.py:3]: Inconsistent indentation. `4*v` and `(math.pi * (d**2)`. Please make it `4 * v`

File: A-set/667A. Pouring Rain.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import math
+d,h,v,e = map(int, input().split())
+drink_s = (4*v)/(math.pi * (d**2))
+if drink_s > e:
```
Comment (Waqar-107) [inline A-set/667A. Pouring Rain.py:4]: Add a new line before this line

**Review by Waqar-107 — CHANGES_REQUESTED** [review]
I have added a few changes related to indentations. Please fix them so that we can merge this pull request. I know there are **old** codes in this repository with bad indentation but let us maintain indentation consistency.

**Comment by Supporter09** [comment]
Done!

File: A-set/667A. Pouring Rain.py [hunk 4]
```diff
@@ -1,9 +1,14 @@
+# from dust i have come, dust i will be
+
 import math
-d,h,v,e = map(int, input().split())
-drink_s = (4*v)/(math.pi * (d**2))
+
+d, h, v, e = map(int, input().split())
+
+drink_s = (4 * v)/(math.pi * (d**2))
```
Comment (Waqar-107) [inline A-set/667A. Pouring Rain.py:7]: `(4 * v) / (math.pi * (d**2))` - there should be spaces before and after the `/`

File: A-set/667A. Pouring Rain.py [hunk 5]
```diff
@@ -1,9 +1,14 @@
+# from dust i have come, dust i will be
```
Comment (Waqar-107) [inline A-set/667A. Pouring Rain.py:1]: no need to add this

File: A-set/667A. Pouring Rain.py [hunk 6]
```diff
@@ -1,9 +1,14 @@
...
-drink_s = (4*v)/(math.pi * (d**2))
+
+d, h, v, e = map(int, input().split())
+
+drink_s = (4 * v)/(math.pi * (d**2))
+
 if drink_s > e:
     print("YES")
     result = (h / (drink_s - e))
     print(round(result,12))
```
Comment (Waqar-107) [inline A-set/667A. Pouring Rain.py:12]: please add a comma after `result`

**Review by Waqar-107 — APPROVED** [review]
Looks good. One friendly suggestion, in the industry, every problems solution is given in a different pull request. Suppose you are implementing a new feature and encounter a bug. You open a new ticket for that bug in the issue tracker and solve it in a different git branch and give a separate pull request.

**Comment by Supporter09** [comment]
Thanks!

**Pull request merged**